.github/workflows/bench.yml

```diff
@@ -37,14 +37,14 @@ jobs:
 
       - name: Run benchmark on current branch
         run: |
-          (for i in {1..${{ steps.settings.outputs.benchmark_repetitions }}}; do go test ./fvm --bench . --tags relic -shuffle=on; done) | tee new.txt
+          (for i in {1..${{ steps.settings.outputs.benchmark_repetitions }}}; do go test ./fvm --bench . --tags relic -shuffle=on --benchmem --run ^$; done) | tee new.txt
```

Comment:
Bench memory and don't run tests (just benchmarks).

Reply:
why not just `-count ${{ steps.settings.outputs.benchmark_repetitions }}`

Reply:
(nit: you may also want to update the "The command `...` was used.")

Reply:
> why not just -count ${{ steps.settings.outputs.benchmark_repetitions }}?

Last time I was checking this, the problem was that using `count + shuffle` didn't do what I wanted. 

count 3 + shuffle of tests {A, B, C} does:
- BBB AAA CCC

I wanted something more spread out, so this does:
- BCA CBA ACB

Which I think is statistically better if the runner is noisy.